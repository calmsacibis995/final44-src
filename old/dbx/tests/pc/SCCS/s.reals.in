h17163
s 00013/00000/00000
d D 5.1 90/06/25 15:37:21 bostic 1 0
c date and time created 90/06/25 15:37:21 by bostic
e
u
U
t
T
I 1
print 3+4
print 3.5+4.5
stop in program
run
next
step
print y
print y + 2
print 3.5*y
stop in f
cont
where
cont
E 1
