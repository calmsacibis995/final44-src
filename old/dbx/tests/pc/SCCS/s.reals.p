h21273
s 00018/00000/00000
d D 5.1 90/06/25 15:37:25 bostic 1 0
c date and time created 90/06/25 15:37:25 by bostic
e
u
U
t
T
I 1
(*
 * Test of reals.
 *)

program realtest (input, output);
var
    x : real;
    y : real;

function f (x : real) : real;
begin
    f := 3.14*x;
end;

begin
    y := 3.0;
    x := f(y);
end.
E 1
