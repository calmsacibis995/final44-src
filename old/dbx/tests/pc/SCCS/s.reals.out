h40612
s 00022/00000/00000
d D 5.1 90/06/25 15:37:24 bostic 1 0
c date and time created 90/06/25 15:37:24 by bostic
e
u
U
t
T
I 1
reading symbolic information ...
7 
8.0 
[1] stop in program
[1] stopped in program at line 15
   15   begin
stopped in program at line 16
   16       y := 3.0;
stopped in program at line 17
   17       x := f(y);
3.0 
5.0 
10.5 
[3] stop in reals.f
[3] stopped in reals.f at line 11
   11   begin
reals.f(x = 3.0), line 11 in "reals.p"
program(), line 17 in "reals.p"
main(0x1, 0x7fffe7e8, 0x7fffe7f0) at 0xad

execution completed

E 1
